install/example-cws-configuration.properties

```diff
@@ -153,6 +153,9 @@ history_days_to_live=7
 # worker table. Allowed values are whole integers 1 <= n
 worker_abandoned_days=1
 
+# worker_max_num_running_procs
+worker_max_num_running_procs=25
```

Comment:
Yeah this is probably too high.  Change it to like 8 as a default.

Reply:
Default set to 16. Added description.